mscore/transposedialog.cpp

```diff
@@ -48,39 +48,47 @@ TransposeDialog::TransposeDialog(QWidget* parent)
       setupUi(this);
       setWindowFlags(this->windowFlags() & ~Qt::WindowContextHelpButtonHint);
 
-      connect(transposeByKey, SIGNAL(clicked(bool)), SLOT(transposeByKeyToggled(bool)));
-      connect(transposeByInterval, SIGNAL(clicked(bool)), SLOT(transposeByIntervalToggled(bool)));
+      connect(transposeByKey,       SIGNAL(clicked(bool)),  SLOT(transposeByKeyToggled(bool)));
+      connect(transposeByInterval,  SIGNAL(clicked(bool)),  SLOT(transposeByIntervalToggled(bool)));
+      connect(chromaticBox,         SIGNAL(clicked(bool)),  SLOT(chromaticBoxToggled(bool)));
+      connect(diatonicBox,          SIGNAL(clicked(bool)),  SLOT(diatonicBoxToggled(bool)));
 
       MuseScore::restoreGeometry(this);
       }
 
 //---------------------------------------------------------
-//   transposeByKeyToggled
+//   TransposeDialog slots
 //---------------------------------------------------------
 
 void TransposeDialog::transposeByKeyToggled(bool val)
       {
       transposeByInterval->setChecked(!val);
       }
 
-//---------------------------------------------------------
-//   transposeByIntervalToggled
-//---------------------------------------------------------
-
 void TransposeDialog::transposeByIntervalToggled(bool val)
       {
       transposeByKey->setChecked(!val);
       }
 
+void TransposeDialog::chromaticBoxToggled(bool val)
+      {
+      diatonicBox->setChecked(!val);
+      }
+
+void TransposeDialog::diatonicBoxToggled(bool val)
+      {
+      chromaticBox->setChecked(!val);
+      }
+
 //---------------------------------------------------------
 //   mode
 //---------------------------------------------------------
 
 TransposeMode TransposeDialog::mode() const
       {
       return chromaticBox->isChecked()
-                  ? (transposeByKey->isChecked() ? TransposeMode::BY_KEY : TransposeMode::BY_INTERVAL)
-                  : TransposeMode::DIATONICALLY;
+         ? (transposeByKey->isChecked() ? TransposeMode::BY_KEY : TransposeMode::BY_INTERVAL)
```

Comment:
Wouldn't basic if statements be easier to read here?

Reply:
I know it's far from important, but since this is a code cleanup PR...

Reply:
I think it's part of the coding principle to only indent 3 spaces if a statement isn't finished.